server/node-postgres-mydb/api/songs.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const router = express.Router();
+const queries = require('../data/queries');
+
+router.get('/', (req, res) =>{
+  queries.getAll().then(song =>{
+    res.json(song);
```

Comment:
I can't work out how you get to see you entire app by going to this endpoint. It should simply be showing json output. The route must be overridden from somewhere else?

Reply:
I used mocha shai super test to test  api song, so have method call getAll and done method and I create my own queries in data folder and requiring the knex configrations